Specs/Number/Korean/NumberModel.json

```diff
@@ -488,23 +502,9 @@
   ]
 }
 ,
-{
-  "Input": "-일만",
```

Comment:
Why was this case removed?
If it should be correct, but currently doesn't work in dotnet, just mark as NotSupported and it will be skipped with a warning.

Reply:
This is because the expression '-일만' is not used in CJK (Chinese, Japanese, Korean).
It also does not exist in 'test Specs' in Chinese or Japanese (like "-一萬")
So I removed that test case.